tests/rptest/services/redpanda.py

```diff
@@ -1317,6 +1321,15 @@ def raise_on_bad_logs(self, allow_list=None):
                             allowed = True
                             break
 
+                if "oversized allocation" in line:
+                    m = re.search("oversized allocation: (\d+) byte", line)
+                    if m and int(m.group(1)) < MAX_ALLOCATION_SIZE:
+                        self.logger.warn(
+                            f"Ignoring oversized allocation, {m.group(1)} is less than the max allowable allocation size of {MAX_ALLOCATION_SIZE} bytes"
+                        )
+                        allowed = True
+                        break
```

Comment:
This needs to be a `continue` otherwise we ignore other potential log lines? Seems broken in the leak sanitizer check as well.

Reply:
`LeakSanitizer` looks like it's doing the right thing: the `break` is in a nested `for` which is re-grepping the entire log (!!) to see if the leak is small enough to ignored, so it should just result in skipping to the next line.

The `break` here def looks like it would silently skip the rest of the lines though, good eye!

Reply:
Nice catch, fixing now